### PR TITLE
Dismiss tooltip/context menus on Escape

### PR DESCRIPTION
**Pull request by mattseddon**
One item from https://github.com/iterative/vscode-dvc/issues/2028:

> - [ ] Dismiss interactive tooltips on ESC (https://github.com/iterative/vscode-dvc/pull/1986#issuecomment-1177989980)

### Demo

https://user-images.githubusercontent.com/37993418/189049026-99192485-7a07-4ebe-a8dc-46a5a69660b9.mov

(trust me I am pressing escape)

### CODE REVIEW

File: webview/src/shared/components/tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -6,6 +6,27 @@ import 'tippy.js/dist/tippy.css'
 export const HEADER_TOOLTIP_DELAY = 100
 export const CELL_TOOLTIP_DELAY = 1000
 
+const hideOnEsc = {
```
Comment (mattseddon) [inline webview/src/shared/components/tooltip/Tooltip.tsx:9]: Found this [here](https://github.com/atomiks/tippyjs/issues/750). It is also shown in the official docs: https://atomiks.github.io/tippyjs/v6/plugins/#hideonesc

Comment (mattseddon) [reply webview/src/shared/components/tooltip/Tooltip.tsx:9]: I can't think of a reason not to have this on any of the tooltips/context menus but LMK if you think differently.

File: webview/src/shared/components/tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -6,6 +6,27 @@ import 'tippy.js/dist/tippy.css'
...
+  defaultValue: true,
+  fn({ hide }: { hide: () => void }) {
+    function onKeyDown(event: KeyboardEvent) {
```
Comment (julieg18) [inline webview/src/shared/components/tooltip/Tooltip.tsx:12]: Things appear to be working correctly for the body cells but the `esc` key isn't working for the header cells on my end:

https://user-images.githubusercontent.com/43496356/189126147-69985ffe-bf05-4a68-8eb7-4f37e5071061.mov

**Review by wolmir — APPROVED** [review]

File: webview/src/shared/components/tooltip/Tooltip.tsx (hunk 2 above)
Comment (mattseddon) [reply webview/src/shared/components/tooltip/Tooltip.tsx:12]: Yep, doesn't work... isn't obvious why either.

Comment (mattseddon) [reply webview/src/shared/components/tooltip/Tooltip.tsx:12]: I'll fix it today.

File: webview/src/experiments/components/table/TableHeaderCellContents.tsx [hunk 3]
```diff
@@ -41,9 +41,6 @@ export const ColumnDragHandle: React.FC<{
     <span
       data-testid="rendered-header"
       className={cx(styles.cellContents)}
-      onKeyDown={e => {
```
Comment (mattseddon) [inline webview/src/experiments/components/table/TableHeaderCellContents.tsx:44]: [F] This was the cause of the breakage in the header context menu. From memory, we put this in to stop tab from causing all kinds of problems before we had a context menu. I could not find any reason to keep it around. 

@wolmir do you remember why you added this?

Comment (wolmir) [reply webview/src/experiments/components/table/TableHeaderCellContents.tsx:44]: @mattseddon No, I can't remember why I added it here

Sorry for the delay in the reply!

**Pull request merged**